tree/ntuple/src/RFieldUtils.cxx

```diff
@@ -273,6 +273,14 @@ void MapIntegerType(std::string &typeName)
    } else if (typeName == "unsigned long long" || typeName == "unsigned long long int") {
       typeName = ROOT::RField<unsigned long long int>::TypeName();
    }
+
+   // The following two types are 64-bit integers on Windows that we can encounter during renormalization of demangled
+   // std::type_info names.
+   if (typeName == "__int64") {
+      typeName = "std::int64_t";
```

Comment:
The other cases uses something like this:
```suggestion
      typeName = ROOT::RField<long long int>::TypeName();
```
if this is not applicable here, could we add a comment explaining why (so that future maintainer know whether this syntax is fine or not).